gateway-shell/src/main/java/org/apache/knox/gateway/shell/ErrorResponse.java

```diff
@@ -28,7 +28,7 @@
     this.response = response;
   }
 
-  public HttpResponse getReponse() {
```

Comment:
This change didn't need require changes anywhere else?

Reply:
Actually, I should remove the printing of the stack trace. The call should fail, but is required to setup the actual underlying request (so the setting of the entity can be verified). Short answer: no, it should not fail the test in either case.

Reply:
No, it couldn't even be used elsewhere because the entire class was package-private before this change.